tests/core/pyspec/eth2spec/test/capella/block_processing/test_process_withdrawals.py

```diff
@@ -601,7 +601,8 @@ def test_success_no_max_effective_balance(spec, state):
     set_eth1_withdrawal_credential_with_balance(spec, state, validator_index, spec.MAX_EFFECTIVE_BALANCE - 1)
     validator = state.validators[validator_index]
 
-    assert validator.effective_balance < spec.MAX_EFFECTIVE_BALANCE
+    # Effective balance should be 1 increment lower than MAX_EFFECTIVE_BALANCE
```

Comment:
In reality it is not only the case. If balance is greater than `X + 0.75 ETH`, where `X` is a round amount of ETH then the EB can either be rounded down or up. But for the test helper it might be fine to have rounding down a default behaviour, the helper can also be extended to set EB explicitly if needed

Reply:
> the helper can also be extended to set EB explicitly if needed

this is what we discussed as well similar to `set_compounding_withdrawal_credential_with_balance`, I was thinking we could round based on `DOWNWARD_THRESHOLD` which would address your concern? but anyhow, being explicit when setting up test data seems favorable to me

Reply:
We can round down based on `DOWNWARD_THRESHOLD` only when `balance + DOWNWARD_THRESHOLD < round_up(balance)`, and, otherwise, round up

Reply:
> but anyhow, being explicit when setting up test data seems favorable to me

Agreed. I think there should be an explicit `effective_balance` parameter here.